library/src/com/anjlab/android/iab/v3/BillingProcessor.java

```diff
@@ -92,6 +100,18 @@ public BillingProcessor(Context context, String licenseKey, IBillingHandler hand
 		contextPackageName = context.getApplicationContext().getPackageName();
 		cachedProducts = new BillingCache(context, MANAGED_PRODUCTS_CACHE_KEY);
 		cachedSubscriptions = new BillingCache(context, SUBSCRIPTIONS_CACHE_KEY);
+		developerMerchantId=null;
+		bindPlayServices();
+	}
+
+	public BillingProcessor(Context context, String licenseKey, String merchantId, IBillingHandler handler) {
```

Comment:
@mustii82 you have to provide your `merchantId` to a new constructor and in case of attack you'll see it in `onBillingError()` handler with special error code